recipes/braceexpand/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "braceexpand" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "13980ad19c80ed6097c245febb694e57baffd8179b50bc80437003eb01465ecb"
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

Comment:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

Reply:
Thanks! According to https://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip `--no-deps` should stay, any opinion?

Reply:
https://github.com/conda-forge/staged-recipes#7-when-or-why-do-i-need-to-use--python---m-pip-install---vv

I think we found one doc not up to date

the ``"{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "`` is the "old" style

Reply:
let the ``--no-deps`` then
it will not hurt
=p